### PR TITLE
CSS for improving group creation UX, namely setting long description

### PR DESCRIPTION
**Pull request by lukebarnard1**
For https://github.com/matrix-org/matrix-react-sdk/pull/1580

### CODE REVIEW

File: src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss [hunk 1]
```diff
@@ -304,6 +304,18 @@ limitations under the License.
 }
 
 .mx_GroupView_groupDesc textarea {
-    width: 50%;
+    width: 100%;
+    max-width: 100%;
```
Comment (ara4n) [inline src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: shouldn't be needed?

File: src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss [hunk 2]
```diff
@@ -304,6 +304,18 @@ limitations under the License.
...
     height: 150px;
 }
+
+.mx_GroupView_groupDesc_placeholder {
+    background-color: $group-my-groups-placeholder-bg;
+    color: $group-my-groups-placeholder-fg;
```
Comment (ara4n) [inline src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: i'm surprised this needs a custom colour? shouldn't one of the existing generic theme colours fit?

File: src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss (hunk 1 above)
Comment (lukebarnard1) [reply src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: It just stops the user being silly and dragging the textarea wider than it should go. But it still allows for them to adjust its vertical height.

File: src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss (hunk 2 above)
Comment (lukebarnard1) [reply src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: There _might_ have been one that already existed but I guess its name felt to specific for me to reuse it somewhere else... although I have contradicted myself here by using a `$group-my-groups-*` variable in GroupView.

Comment (ara4n) [reply src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: e.g. the "preview-bg-color" or whatever it's called.  if it feels icky to reuse the same constant for this component, one could always rename it as "info-plinth-bg-color" or something

**Comment by ara4n** [comment]
lgtm mod comments

File: src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss (hunk 2 above)
Comment (lukebarnard1) [reply src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: `info-plinth-bg-color` +1 !

File: src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss (hunk 1 above)
Comment (ara4n) [reply src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss]: "it's textareas being weird" i think is the answer then :D

**Pull request merged**